checkov/common/bridgecrew/integration_features/features/suppressions_integration.py

```diff
@@ -70,7 +71,12 @@ def _apply_suppressions_to_report(self, scan_report: Report) -> None:
         still_failed_checks = []
         still_passed_checks = []
         for check in scan_report.failed_checks + scan_report.passed_checks:
-            relevant_suppressions = self.suppressions.get(check.check_id)
+            check_id = check.check_id
```

Comment:
I suggest to put the next logic under a method (e.g adjust_check_id) + adding doc why we have to do it + test this new small method for understanding what is expected original check_id